web/src/app/modules/views/main/authentication/modules/login/components/login.component.html

```diff
@@ -60,6 +60,21 @@ <h1 class="card-title text-center mb-4 mt-1">{{'login' | translate}}</h1>
             </button>
             <span id="login-language-dropdown" class="pull-right"><language-chooser></language-chooser></span>
         </div>
+        <div class="form-group">
+              <!----<img  height="200" *ngFor="let url of urls" [src]="url">-->
+<input type='file' (change)="onSelectFile($event)">
```

Comment:
Why is here an input field? For variant 1 of this feature, the user shall not be able to define the background image.